.github/workflows/test.yml

```diff
@@ -18,6 +18,10 @@ jobs:
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
```

Comment:
I wish we could somehow sync this up with the `.nvmrc` file at some point 🤔

Reply:
That could be handy, the main reason I added this to the test.yml is in case the default on the system changes.